app/models/lookup_keys/lookup_key.rb

```diff
@@ -203,6 +204,16 @@ def cast_default_value
     true
   end
 
+  def reset_override_params
+    if override_changed? && !override?
```

Comment:
I don't see a huge benefit to doing so but if you think it's better, then I can.